dev-docs/modules/haloRtdProvider.md

```diff
@@ -16,21 +16,24 @@ sidebarType : 1
 * TOC
 {:toc}
 
-Audigent is a next-generation data management platform and a first-of-a-kind
-"data agency" containing some of the most exclusive content-consuming audiences
-across desktop, mobile and social platforms.
-
-This real-time data module provides an integrated post-cookie Halo identity and
-real-time user segmentation solution that seamlessly integrates into your bid
-request cycle. Users are identified and bid request objects are augmented
-with first-party data derived segments prior to being sent to exchange/ssp
-services in order to optimize targeting and increase publisher revenue.
-
-Audigent maintains a large database of first-party Tradedesk Unified ID,
-Audigent Halo ID and other id provider mappings to various third-party segment
-types that are utilizable across different backends. With the Halo RTD module,
-these segments and other data can be retrieved and utilized by supporting
-exchange and SSP backends in real-time during your bid request cycle.
+Audigent is a next-generation, 1st party data management platform and the
+world’s first "data agency", powering the programmatic landscape and DTC
+eCommerce with actionable 1st party audience and contextual data from the
+world’s most influential retailers, lifestyle publishers, content creators,
+athletes and artists.
+
+The Halo real-time data module in Prebid has been built so that publishers
+can maximize the power of their first-party audiences and contextual data.
+This module provides both an integrated cookieless Halo identity with real-time
+contextual and audience segmentation solution that seamlessly and easily
+integrates into your existing Prebid deployment.
+
+Users, devices, content, cohorts and other features are identified and utilized
+to augment every bid request with targeted, 1st party data-derived segments
+before being submitted to supply-side platforms. Enriching the bid request with
+robust 1st party audience and contextual data, Audigent's Halo RTD module
+optimizes targeting, increases the number of bids, increases bid value,
+and drives additional incremental revenue for publishers.
```

Comment:
Again, please take out any language that implies a better or improved performance on bids or revenue. A solution if you'd like to promote your product is to rewrite the text something like so: 

before being submitted to supply-side platforms. Enriching the bid request with
robust first-party audience and contextual data, Audigent's Halo RTD module
helps optimize targeting and performance. See our detailed performance reviews here...

and link to your site...